### DISCUSSION TITLE
Does building with twin generate a separate CSS file ?

### DISCUSSION THREAD

**Discussion by VaZark** [ben-rogerson/twin.macro#742] [Q&A]
I'm currently building an embeddable widget with custom element which uses tailwind. When building with parcel, I end up creating a JS and a CSS file that needs to be imported by the client. 

Can twin allow me generate a single JS file with the CSS embedded within?

**Comment by ben-rogerson** [comment]
You should look at Parcel settings as it's doing the building - I'd help but haven't tried to do that before.
